### ISSUE TITLE
Change Sounds

### ISSUE THREAD

**Issue by rooftopbbq** [stijnwop/tirePressure#7]
Is it possible to change the sounds for inflate and deflate qith custom sounds?

**Comment by stijnwop** [comment]
Yes sure, just load them like you do with any other sound. Nothing different here.

**Comment by rooftopbbq** [comment]
OK. Perfect. Will try it.
Thank you
